app-admin/lnav/lnav-0.10.0-r1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
I recommend renaming this file to `lnav-0.10.0.ebuild` since this is the first edition of the ebuild.